notebooks/book2/14/logreg_iris_bayes_2d_pymc.ipynb

```diff
@@ -0,0 +1,356 @@
+{
```

Comment:
<p>In original figure there is two plots, Can I know why I'm able to see only one plot here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Bhaiya this is because the plot is controlled by an unbalanced (variable in the code). Based on its value one of the two plot is generated.

Reply:
You can take the same height for both plots by taking min(plot1_height, plot2_height)